config_defaults_inc.php

```diff
@@ -2828,6 +2828,15 @@
  */
 $g_create_project_threshold = ADMINISTRATOR;
 
+/**
+ * Threshold needed to create a subproject
+ *
+ * Edit is controlled by $g_manage_project_threshold
+ *
+ * @global integer $g_create_subproject_threshold
+ */
+$g_create_subproject_threshold = MANAGER;
```

Comment:
I would use the same default as `create_project_threshold` for backward compatibility.